schema/common.json

```diff
@@ -30,45 +30,7 @@
       ]
     },
     "entry": {
-      "title": "Entry",
-      "type": "object",
-      "additionalProperties": false,
-      "properties": {
-        "message": {
-          "description": "A human-readable one-line description of the entry",
-          "type": "string",
-          "examples": ["Point is not writable"]
-        },
-        "detail": {
-          "description": "An optional extensive entry which can include more detail, e.g. a complete program stack-trace",
-          "type": "string"
-        },
-        "category": {
-          "$ref": "file:category.json"
-        },
-        "timestamp": {
-          "description": "Timestamp the condition was triggered, or most recently updated",
-          "type": "string",
-          "format": "date-time",
-          "examples": ["2018-08-26T21:39:28.364Z"]
-
-        },
-        "level": {
-          "$comment": "https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#logseverity",
-          "description": "The status `level` should conform to the numerical [Stackdriver LogEntry](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#logseverity) levels. The `DEFAULT` value of 0 is not allowed (lowest value is 100, maximum 800).",
-          "type": "integer",
-          "multipleOf": 1,
-          "minimum": 100,
-          "maximum": 800,
-          "examples": [600]
-        }
-      },
-      "required": [
-        "message",
-        "category",
-        "timestamp",
-        "level"
-      ]
+      "$ref": "file:entry.json"
```

Comment:
this seems like an unnecessary redirection for all the existing uses -- does a s/x/y replace work to update existing uses to not have this reference in common.json?

Reply:
I think this $ref vs inline decl is equivalent depth? i.e. field access is still entry.message.

Reply:
It introduces an unnecessary redirection, and multiple ways of specifying the same thing. Other places have

` "$ref": "file:common.json#/definitions/entry"`

which then maps directly to 

`"$ref": "file:entry.json"`

I see no reason to keep the original "#/definitions/entry" version around.

Reply:
Oh I get you now. Done